config/ftbquests/quests/chapters/botania.snbt

```diff
@@ -410,12 +414,18 @@
 				{
 					id: "0000000000000523",
 					type: "item",
+					title: "Open Crate",
 					item: "botania:open_crate"
 				},
 				{
 					id: "00000000000005E8",
 					type: "item",
-					item: "botania:monocle"

```

Comment:
Only section that looks odd to me is this one. I don't recall adding these other lines, just the title. Perhaps this is OK and FTB Quests simply added this data in automagically?

Reply:
Github isn't showing the lines I wanted it to, but if you View Changes and search for "botania:monocle", you'll see the section I'm talking about.

Reply:
I saw this change on the last PR, I think you're right that FTBQ adds it automagically :) all good.